decidim-meetings/app/commands/decidim/meetings/admin/copy_meeting.rb

```diff
@@ -88,7 +88,7 @@ def fields_from_meeting
         def copy_services!
           form.services_to_persist.map do |service|
             Decidim::Meetings::Service.create!(
-              meeting: copied_meeting,
+              :meeting => copied_meeting,
```

Comment:
We could use just the new syntax here.

Reply:
Done in 984ace2000ad97e768bb8eba091d4ba7eda2ac11